src/pkg/services/m365/m365_test.go

```diff
@@ -473,6 +497,45 @@ func (suite *DiscoveryIntgSuite) TestSites_InvalidCredentials() {
 	}
 }
 
+func (suite *DiscoveryIntgSuite) TestTeams_InvalidCredentials() {
```

Comment:
`DiscoveryIntgSuite` is an old artifact.  We should probably get rid of it.  In the meantime, can you please move this test into the other suite?